cf-oscore/src/main/java/org/eclipse/californium/oscore/OSCoreUdpStack.java

```diff
@@ -60,19 +62,19 @@ public OSCoreStack(String tag, Configuration config, EndpointContextMatcher matc
 	}
 
 	/**
-	 * Creates a new stack for UDP as the transport.
+	 * Creates a new OSCORE-enabled stack using UDP as the transport.
 	 * 
 	 * @param tag logging tag
 	 * @param config The configuration values to use.
 	 * @param outbox The adapter for submitting outbound messages to the
 	 *            transport.
 	 * @param ctxDb context DB.
 	 * @deprecated use
-	 *             {@link #OSCoreStack(String, Configuration, EndpointContextMatcher, Outbox, OSCoreCtxDB)}
+	 *             {@link #OSCoreUdpStack(String, Configuration, EndpointContextMatcher, Outbox, OSCoreCtxDB)}
 	 *             instead
 	 * @since 3.0 (logging tag added and changed parameter to Configuration)
 	 */
-	public OSCoreStack(String tag, Configuration config, Outbox outbox, OSCoreCtxDB ctxDb) {
+	public OSCoreUdpStack(String tag, Configuration config, Outbox outbox, OSCoreCtxDB ctxDb) {
```

Comment:
Now it's a new class, so we don't need the deprecated constructor.
With the new name, it's anyway required to adapt the using code. With that, the backwards compatibility is broken, and so wen don't benefit from providing the deprecated variant.

Reply:
Done in commit 2861fb22ba8d37687159909b017663b15631fded